extensions/splunk/PutSplunkHTTP.h

```diff
@@ -40,7 +40,23 @@ class PutSplunkHTTP final : public SplunkHECProcessor {
   PutSplunkHTTP& operator=(PutSplunkHTTP&&) = delete;
   ~PutSplunkHTTP() override = default;
 
-  EXTENSIONAPI static constexpr const char* Description = "Sends the flow file contents to the specified Splunk HTTP Event Collector over HTTP or HTTPS. Supports HEC Index Acknowledgement.";
+  EXTENSIONAPI static constexpr const char* Description =
+      "Sends the flow file contents to the specified Splunk HTTP Event Collector (see https://docs.splunk.com/Documentation/SplunkCloud/latest/Data/UsetheHTTPEventCollector) over HTTP or HTTPS.\n"
+      "\n"
+      "The \"Source\", \"Source Type\", \"Host\" and \"Index\" properties are optional and will be set by Splunk if unspecified. If set,\n"
```

Comment:
Are the `\n` endline characters necessary to wrap the descriptions? Isn't the wrapping done automatically in the markdown files?

Reply:
I think it's useful if `PROCESSORS.md` can be read in both a markdown reader (eg. the GitHub UI) and as a plain text file (eg. in `less` or `vim`).  Newlines are useful in the second case, and harmless in the first case.

Reply:
Sounds reasonable, thanks for elaborating!